### ISSUE TITLE
Extract shortcuts from Hail

### ISSUE THREAD

**Issue by rahaaatul** [MM2-0/Kvaesitso#335]
Option to extract shortcuts from [Hail](https://github.com/aistra0528/Hail) app (that are disabled) to the Kvaesitso's app drawer/widget/search.

**Comment by rahaaatul** [comment]
Hail is an app disabler app with the help of shizuku. It creates shortcuts in home to enable and open the app. I'm asking kvaesitso to get all those shortcut and show them as app in the search/drawer/widget.

**Comment by MM2-0** [comment]
Shortcuts should appear in the favorites section. I still don't fully understand what your requets is. Is it to have shortcuts in the app grid instead of the favorites?

**Comment by rahaaatul** [comment]
Yes, yes, exactly what I'm requesting! Hail shortcuts that appear in the app grid. But appearing in the favorites is fine too.

Also the shortcut doesn't get the correct icon when customizing the icon.
![Screenshot_20230423_195224](https://user-images.githubusercontent.com/86149861/233843813-4ed98c52-1873-4387-a525-1672393eeec1.jpg)
![Screenshot_20230423_195234](https://user-images.githubusercontent.com/86149861/233843816-efd36531-e89d-41d6-b23f-0987bad3e9ba.jpg)

By the way, this launcher gets slow when I open the search bar, is there a way to fix this problem? Thank you.

**Comment by MM2-0** [comment]
> Yes, yes, exactly what I'm requesting! Hail shortcuts that appear in the app grid. But appearing in the favorites is fine too.

Similar to #133 then?

> Also the shortcut doesn't get the correct icon when customizing the icon.

There is no way the launcher can now which app this shortcut is supposed to represent. All it knows is which app created the shortcut. So you have to search for the icon manually.

**Comment by rahaaatul** [comment]
> Similar to #133 then?
Almost. But I don't want to add them manually. My idea was, Kvaesitso will add them automatically.

> There is no way the launcher can now which app this shortcut is supposed to represent. All it knows is which app created the shortcut. So you have to search for the icon manually.
Then how does Niagara do it?

https://user-images.githubusercontent.com/86149861/233845867-a3132bd3-47ae-4587-b240-e758ea89d352.mp4

**Comment by MM2-0** [comment]
> Then how does Niagara do it?

Huh. I guess I was wrong.
